requirements.in

```diff
@@ -0,0 +1,5 @@
+semantic_version
+pytz
+datetime
+flask
+requests
```

Comment:
@GeetikaBatra Please double-check these requirements. It seems to me that some of them are not really needed.

Reply:
These will be needed for future use, once Yusuf integrates his changes.

Reply:
Isn't this supposed to be a cron job? Do we really need Flask? If yes, then I am very confused about the implementation :)

Reply:
@msrb umm yeah, you are right. Removing `flask` :)